.gitignore

```diff
@@ -22,7 +22,9 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
-
+/config/credentials.yml.enc
```

Comment:
You added this file to .gitignore but it is still included in repo. 

If file already exists and you add it to .gitignore then it is still in repo. You need to remove from repo manually.

The question we should discuss is do we want to use this rails credentials solution or just work on ENV variables. We can discuss it during the next sprint call